common-content-dataflow/modules/ROOT/partials/job-specification.adoc

```diff
@@ -272,15 +272,15 @@ The options `key_constraints` and `unique_constraints` require Neo4j/Aura Enterp
   "active": true,
   "write_mode": "merge",
   "source_transformations": {
-    "enable_grouping": true  // is this on by defult?
+    "enable_grouping": true
```

Comment:
My comment meant: if it's `true` by default, we can avoid specifying it and alter the description below saying "unless explicitly set to false, ...", so we don't draw attention to features that are enabled by default (assuming we've chosen so because most people will want them on).

Reply:
it is not `true` by default ^^